### PR TITLE
Fix download directory setting and other improvements

### LINKED ISSUE #47: Change download folder doesn't have affect
Cant switch the download Path.

If u change the download folder to a specific path, downloads wont shown in this and are hidden on the cloud storage. 
After switching back to the app standard download path, surprise, all downloads will shown in this folder.

So ncdownloader cant handle other path's than the regular path which is used after install the app. No changes has affect. 

Hope to see a fix for it. :-) 

Greatz Paddy

### PR DESCRIPTION
**Pull request by polter-rnd**
- Fix #47 (some refactoring to download directory management)
- Fix `make dist` command (exclude files while packaging)
- Update the interface after download was started

### CODE REVIEW

**Review by shiningw — APPROVED** [review]

File: src/App.vue [hunk 1]
```diff
@@ -24,10 +24,15 @@ const successCallback = (data, element) => {
...
-    helper.message(t("ncdownloader", "Downloading" + " " + data.file));
+  } else {
+    if (data.hasOwnProperty("message")) {
+      helper.message(t("ncdownloader", data.message));
+    } else if (data.hasOwnProperty("file")) {
+      helper.message(t("ncdownloader", "Downloading" + " " + data.file));
+    }
+
+    document.getElementById("text-input-value").value = "";
+    document.querySelector(".active-downloads a").click();
```
Comment (polter-rnd) [inline src/App.vue:35]: Not sure if it is the proper way to do this, though

Comment (shiningw) [reply src/App.vue:35]: What do you want to achieve here? start polling immediately after loading the main page?

Comment (polter-rnd) [reply src/App.vue:35]: Want to start polling right after the download was started. Without it "Active Downloads" count donesn't update immediately after starting the download

Comment (shiningw) [reply src/App.vue:35]: for this, you just need to call  `helper.loop(helper.refresh, delay, ...["/apps/ncdownloader/status/active"])`

but I don't think we need this as you will eventually click on the tab if you'd like to check your download progress

**Pull request closed without merging**